frontend/src/features/admin-form/settings/components/EndPageSettingsSection/EndPageSettingsInput.tsx

```diff
@@ -0,0 +1,111 @@
+import { useCallback, useMemo } from 'react'
```

Comment:
suggest reclarifying with designers on whether we should save values on blur for this screen, i think design may not be updated.

i think we moved on to a submit button for saving instead of autosaving for group inputs.

if we have a submit button on this component, we can remove all the onBlur calls (which looks dangerous to me as they could hold old state for other parts of the endpage inputs)

Reply:
Okay, checking with the designers to use a submit button sintead